src/Maestro/DependencyUpdater/DependencyUpdater.cs

```diff
@@ -207,55 +207,62 @@ select new
             }
         }
 
-        [CronSchedule("0 0 0 1/1 * ? *", TimeZones.PST)]
+        [CronSchedule("0 0/1 * * * ? *", TimeZones.PST)]
```

Comment:
This seems excessive 😮  (I realize this was you debugging)

Reply:
OOOOOOOOOOOOPS